itests/integrations/main.java

```diff
@@ -1,5 +1,5 @@
 ///usr/bin/env jbang "$0" "$@" ; exit $?
-
+//JAVA 13+
```

Comment:
Just curious, why this change?

Reply:
Oh, probably something to do with CDS?

Reply:
yeah, im not sure why it didnt happen in normal testing but without it it complained about CDS not being available before java 13...